.github/workflows/generate-instance.yml

```diff
@@ -0,0 +1,50 @@
+name: Generate Cred Instance
+on:
+  # Trigger on merging to master.
+  push:
+    branches:
+      - master
+  # As well as every 6 hours.
+  schedule:
+    - cron: 0 */6 * * *
+
+jobs:
+  GenerateCredInstance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false # Required to make github pages deployment work correctly
+
+      - name: Cache Data # Cache SourceCred Data, invalidating if any of the config changes or the SC version is updated
+        uses: actions/cache@v2
+        with:
+          path: '**/cache'
+          key: SC-${{ runner.os }}-${{ hashFiles('**/config.json', '**/sourcecred.json', '**/yarn.lock') }}
```

Comment:
Is there a way to invalidate the cache after an age limit, e.g. 7 days? Or would we need to do this on the plugin level?

Reply:
Kinda, we could do something like this to invalidate the cache every Sunday:
```yml
  - name: Get Week
    id: get-week
    run: echo "::set-output name=week::$(/bin/date -u "+%Y-%U")"
    shell: bash
  - name: Cache Data
    uses: actions/cache@v2
    with:
      path: '**/cache'
      key: SC-${{ runner.os }}-${{ steps.get-week.outputs.week }}-${{ hashFiles('**/config.json', '**/sourcecred.json', '**/yarn.lock') }}
```